careless/stats/ccanom.py

```diff
@@ -0,0 +1,133 @@
+"""
+Compute CCanom from careless output.
+"""
+import argparse
+import matplotlib.pyplot as plt
+import reciprocalspaceship as rs
+import seaborn as sns
+
+
+class ArgumentParser(argparse.ArgumentParser):
+    def __init__(self):
+        super().__init__(
+            formatter_class=argparse.RawTextHelpFormatter, 
+            description=__doc__
+        )
+
+        # Required arguments
+        self.add_argument(
+            "mtz",
+            nargs="+",
+            help="MTZs containing crossvalidation data from careless",
+        )
+
+        # Optional arguments
+        self.add_argument(
+            "-m",
+            "--method",
+            default="spearman",
+            choices=["spearman", "pearson"],
+            help=("Method for computing correlation coefficient (spearman or pearson)"),
+        )
+
+        self.add_argument(
+            "-b",
+            "--bins",
+            default=12,
```

Comment:
It's a little funny to me that the default here is `12`, but the functions themselves have a default of `10`. This is fine to leave as is though -- up to you.

Reply:
haha. it's a little silly, but i like 12 for no real reason.

Reply:
Let's do 10 as a default, for my sanity. I've used 10 so far throughout.

Reply:
(I'm sure you can find an example where I did not use 10 but that's besides the point).

Reply:
okay i'll change it to 10.

Reply:
sorry to be a stickler, but the help strings still say the default is 12